versions.properties

```diff
@@ -3,7 +3,7 @@
 ## Dependencies and Plugin versions with their available updates
 ## Generated by $ ./gradlew refreshVersions
 ## Please, don't put extra comments in that file yet, keeping them is not supported yet.
-plugin.android=4.1.1
+plugin.android=4.2.0-beta04
```

Comment:
Compose requires AGP 4.2.0 at least.

Reply:
This kind of thing is why you kept a separate repo earlier. So glad that's done.

Reply:
True, but that was only _required_ because it was using a fork of kotlin itself. We've used pre-release AGP versions before in this project, and I've been using the 4.2.0 canaries forever so I'm really not worried.